app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,12 @@
         android:supportsRtl="true"
         android:theme="@style/Theme.FirebaseTutorial"
         tools:targetApi="31">
-
+        <activity
+            android:name=".HomeActivity"
+            android:exported="false" />
+        <activity
```

Comment:
Maybe change the login activity as the root ?

Reply:
Shall change later